lib/ecto/migration.ex

```diff
@@ -658,4 +659,17 @@ defmodule Ecto.Migration do
   defp validate_type!(%Reference{} = reference) do
     reference
   end
+
+  def prefix(opts_prefix) when is_nil(opts_prefix) do
```

Comment:
This is imported with "use" and fails in migrator_test.exs if they are private (although, they could be private aside from that.  should I remake the prefix functions within the test, or leave them public?